python/lsst/ts/scriptqueue/script_queue.py

```diff
@@ -35,7 +35,7 @@
 and the maximum is SCRIPT_INDEX_MULT-1 more.
 """
 _MAX_SCRIPTQUEUE_INDEX = salobj.MAX_SAL_INDEX//SCRIPT_INDEX_MULT - 1
-_LOAD_TIMEOUT = 20  # seconds
+_LOAD_TIMEOUT = 60  # seconds
```

Comment:
This variable is not used (only the version in script_info is used). Could you please delete this line instead of changing the value?